src/app/containers/PageContainer/components/DefaultHeaderComponent/DefaultHeaderComponent.tsx

```diff
@@ -284,244 +309,234 @@ export const DefaultHeaderComponent: React.FC = () => {
+                      href={bitocracyUrl}
+                      hrefExternal
+                      text={t(translations.mainMenu.voting)}
+                      label={t(translations.mainMenu.labels.voting)}
+                      data-action-id="header-bitocracy-link-voting"
+                    />
+                    <MenuItem
+                      href="https://forum.sovryn.app/"
+                      hrefExternal
+                      text={t(translations.mainMenu.forum)}
+                      label={t(translations.mainMenu.labels.forum)}
+                      data-action-id="header-bitocracy-link-forum"
+                    />
+                  </Menu>
                 }
-              />
+              >
+                <span className={styles.firstLevelMenu}>
+                  <span
+                    className={styles.headerText}
+                    data-action-id="header-link-bitocracy"
+                  >
+                    {t(translations.mainMenu.bitocracy)}
+                  </span>
+                  <FontAwesomeIcon icon={faCaretDown} size="xs" />
+                </span>
+              </NavPopover>
             </Menu>
           </div>
         </div>
 
         <Menu className="tw-flex tw-justify-start tw-items-center">
-          <li>
-            <Link
-              to={{
-                pathname: '/wallet',
-              }}
-              className={classNames(
-                'tw-btn-action tw-text-primary hover:tw-bg-transparent tw-hidden sm:tw-flex tw-mx-2 2xl:tw-mr-4',
-                {
-                  'tw-hidden': !connected,
-                },
-              )}
-              data-action-id="header-link-deposit"
+          <li className="2xl:tw-mr-8">
+            <Popover
+              interactionKind={PopoverInteractionKind.HOVER}
+              position={PopoverPosition.BOTTOM}
+              className="tw-pl-4"
+              popoverClassName="tw-mw-340"
+              content={
+                <div className="tw-px-3.5 tw-py-2 tw-font-normal">
+                  <div className="tw-mb-6">
+                    {t(translations.mainMenu.tooltips.fundWalletButton)}
+                  </div>
+                  <a
+                    href="https://wiki.sovryn.app/en/getting-started"
+                    target="_blank"
+                    rel="noreferrer"
+                  >
+                    {t(translations.mainMenu.tooltips.readMore)}
+                  </a>
+                </div>
+              }
             >
-              {t(translations.common.deposit)}
-            </Link>
+              <Button
+                text={t(
+                  translations.mainMenu[hasFunds ? 'fundWallet' : 'getStarted'],
+                )}
+                href="/fast-btc/deposit"
+                type={ButtonType.button}
+                color={ButtonColor.primary}
+                style={ButtonStyle.normal}
+                size={ButtonSize.md}
+                disabled={!connected}
```

Comment:
Was still able to click on this button when wallet is not connected (maybe because it become link when href was provided and not an actual button)

Reply:
Good catch, it looked disabled but that was just styling :D